configure.ac

```diff
@@ -3,7 +3,7 @@ AC_PREREQ([2.60])
 define(_CLIENT_VERSION_MAJOR, 2)
 define(_CLIENT_VERSION_MINOR, 0)
 define(_CLIENT_VERSION_REVISION, 0)
-define(_CLIENT_VERSION_BUILD, 0)
+define(_CLIENT_VERSION_BUILD, 1)
```

Comment:
Please count up (_CLIENT_VERSION_REVISION) insted of (_CLIENT_VERSION_BUILD).
As build versions are often omitted,it is not the best